<ui version="4.0" >
 <class>QgsRendererV2PropsDialogBase</class>
 <widget class="QDialog" name="QgsRendererV2PropsDialogBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Renderer V2 properties</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Renderer:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radSingleSymbol" >
       <property name="text" >
        <string>single symbol</string>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radCategorized" >
       <property name="text" >
        <string>categorized</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radGraduated" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="text" >
        <string>graduated</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>111</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSymbolLevels" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>Symbol levels...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <widget class="QStackedWidget" name="stackedWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="pageSingleSymbol" >
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Symbol:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QPushButton" name="btnChangeSingleSymbol" >
         <property name="text" >
          <string>Change</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="2" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>271</width>
           <height>181</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageCategorized" >
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>Column:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2" >
        <widget class="QComboBox" name="cboCategorizedColumn" />
       </item>
       <item row="1" column="0" colspan="6" >
        <widget class="QTreeView" name="viewCategories" >
         <property name="iconSize" >
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="rootIsDecorated" >
          <bool>false</bool>
         </property>
         <property name="allColumnsShowFocus" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QPushButton" name="btnAddCategories" >
         <property name="text" >
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" >
        <widget class="QPushButton" name="btnDeleteCategory" >
         <property name="text" >
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3" >
        <widget class="QPushButton" name="btnDeleteAllCategories" >
         <property name="text" >
          <string>Delete all</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="5" >
        <widget class="QPushButton" name="btnJoinCategories" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>Join</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageGraduated" >
      <layout class="QVBoxLayout" >
       <item>
        <layout class="QGridLayout" >
         <item row="0" column="0" >
          <widget class="QLabel" name="label_4" >
           <property name="text" >
            <string>Column:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2" >
          <widget class="QComboBox" name="cboGraduatedColumn" />
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="label_6" >
           <property name="text" >
            <string>Symbol:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QPushButton" name="btnChangeGraduatedSymbol" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>change</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3" >
          <widget class="QLabel" name="label_5" >
           <property name="text" >
            <string>Classes:</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="4" >
          <widget class="QSpinBox" name="spinGraduatedClasses" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="value" >
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="label_7" >
           <property name="text" >
            <string>Color ramp:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="QComboBox" name="cboGraduatedColorRamp" />
         </item>
         <item row="2" column="3" >
          <widget class="QLabel" name="label_8" >
           <property name="text" >
            <string>Mode:</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="4" >
          <widget class="QComboBox" name="cboGraduatedMode" >
           <item>
            <property name="text" >
             <string>Equal Interval</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Quantile</string>
            </property>
           </item>
          </widget>
         </item>
         <item rowspan="2" row="1" column="2" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeView" name="viewGraduated" />
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QPushButton" name="btnGraduatedClassify" >
           <property name="text" >
            <string>Classify</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnGraduatedDelete" >
           <property name="text" >
            <string>Delete class</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QgsRendererV2PropsDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QgsRendererV2PropsDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
